spec/support/helpers/workflow_states.rb

```diff
@@ -20,14 +21,18 @@ def self.previous_state(transient_registration)
       state_machine = transient_registration.class.aasm
 
       state_machine.events.each do |event|
```

Comment:
I'd love to get some comments on this method! Otherwise it does require some digging into the AASM API to understand what is going on.